### ISSUE TITLE
Policies for Subgroups

### ISSUE THREAD

**Issue by SCKelemen** [wearehx/policies#12]
I think a discussion is also in order for the policies of subgroups. 
Subgroups are affiliated with the main group, HX. However, I don't think we should be telling subgroup admins how to govern their group, or what content is allowed or not allowed. 

HX should give the subgroup admins tools to help them perform their duties, and maintain a high quality, content driven, group and community. This, in turn, will have positive effect on HX as a whole. 

Consider HX as the UN, and subgroups as the member countries. We do not need to enforce a policy for each subgroup on how administration will work, but rather we can give the admins tools to help them do a better job, and give the members of the groups, ie citizens, a second place to seek help, when its needed.

**Comment by gsingh93** [comment]
I see no valid reason _against_ having subgroups.

Let's look at HH subgroups. Were HH admins involved _at all_ with the moderation/administration of HH subgroups? As far as I know, no. HH subgroups had complete control over how their groups were run, almost as if they weren't affiliated with HH at all.

If that's the case, why have subgroups in the first place. There are two benefits I see. First of all, it's not even a question whether those groups would have grown to the size they did if the were not affiliated with HH. People normally would not have looked for or accidentally discovered their favorite communities if they were not associated with HH. The fact that most people were in multiple HH communities also meant they'd see the same familiar faces across these groups. I've made some great friends in these groups, and I can guarantee if these were not HH groups I would either not have found them or would not have joined them.

The second benefit I see is it's a filter for content for the main HH subgroup. For example, I love to talk about Rust, but I can understand that tons of people don't care at all. I don't care about JavaScript and PHP, and I'd love to not have to see any content related to them. Subgroups help achieve this, so each user can "subscribe" to the content they want when joining the group.

If that's the case though, what's the point of the main group? There's still tons of content that's applicable to the broad audience that doesn't fit in a subgroup, and there a bunch of examples of this in the group already. There's also some content that can overlap between fitting in a subgroup and fitting in the main group, and I think posting in both places is fine. An example of this type of content is security news, where it's often good to have some type of PSA post in the main group, but you can have a more detailed discussion in a Information Security group.

So to summarize, having a subgroups has a ton of upsides and almost no downsides. There's some unwarranted fear about being associated with a bigger community in case things go to shit like HH, but even after (and while) things went to shit in HH, my subgroup communities were awesome. Literally all you get (and all you need) from the main community association is more publicity and more members, which is a net positive for the group.

**Comment by SCKelemen** [comment]
I agree. Subs help me find content, and help me filter the things I want to see. I mostly used HH to find other subs.

**Comment by arirawr** [comment]
I don't think we have enough content yet to start siphoning it off. There's already very few posts, so we don't want to push the few that exist away.

**Comment by arirawr** [comment]
I agree that HH subgroup communities are awesome, but we want to make _our_ community awesome first. The reason subgroup awesomeness stands out is because HH didn't have that feeling anymore, and ideally we want that feeling in the actual main group

**Comment by SCKelemen** [comment]
I think it's just hard to justify good contain for such a wide topic?
I totally see your point, though.

**Comment by iangcarroll** [comment]
I support the idea of (independent) subgroups. It gives them freedom in moderation while being connected to HX, it requires little work on our part to maintain and create, does not significantly take away from the main group, and if existing subgroups migrated, they would bring in new members to HX.

I agree with everything in @gsingh93's comment as well.

**Comment by SCKelemen** [comment]
I think we can close this, yes? @iangcarroll

**Comment by gsingh93** [comment]
Probably want to put something in the administration policy first.

**Comment by SCKelemen** [comment]
Are we still seeking comments on this?
